src/Kaleidoscope-HardwareTestMode.h

```diff
@@ -0,0 +1,19 @@
+/* Kaleidoscope-Model01-TestMode - A factory test mode for the Model 01.
+ * Copyright (C) 2017-2018  Keyboard.io, Inc.
```

Comment:
I guess this part of the header could use an update? A few other files would need the name & copyright dates updated too.

Reply:
force-pushed a cleanup of this.